app/code/Magento/Theme/Block/Html/Header.php

```diff
@@ -14,6 +16,27 @@
  */
 class Header extends \Magento\Framework\View\Element\Template
 {
+    /**
+     * @var Escaper
+     */
+    protected $escaper;
+
+    /**
+     * Constructor
+     *
+     * @param \Magento\Framework\View\Element\Template\Context $context
+     * @param Magento\Framework\Escaper $escaper
+     * @param array $data
+     */
+    public function __construct(
+        \Magento\Framework\View\Element\Template\Context $context,
+        \Magento\Framework\Escaper $escaper,
+        array $data = []
+    ) {
+        $this->_escaper = $escaper;
```

Comment:
Property declared at line 22 don't has underscore.
Please don't use underscored variables and properties.